Dependencies.lua

```diff
@@ -20,6 +20,7 @@ LibraryDir = {}
 
 LibraryDir["VulkanSDK"] = "%{VULKAN_SDK}/Lib"
 LibraryDir["VulkanSDK_Debug"] = "%{wks.location}/Hazel/vendor/VulkanSDK/Lib"
+LibraryDir["VulkanSDK_DLL"] = "%{wks.location}/Hazel/vendor/VulkanSDK/Bin"
```

Comment:
Might be better named VulkanSDK_DebugDLL   (assuming the post build step only needs to be done for DEBUG build)